legal-api/src/legal_api/services/authz.py

```diff
@@ -164,24 +166,21 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
                 }
             },
             'courtOrder': {
-                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
-                'blockerChecks': {
-                    'business': [BusinessBlocker.DEFAULT]
-                }
+                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC']
             },
             'dissolution': {
                 'voluntary': {
-                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC'],
                     'blockerChecks': {
                         'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
```

Comment:
As SP/GP has been removed from voluntary dissolutions now, we can remove `'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],`